gerencio/gerencio-upgrade-v3
# Resolve vulnerabilities in dependencies

## Status

- [COMPLETE]

## Todos

- [x] - add resolutions

## Deploy notes

- Nothing

## Steps to Test or Reproduce

1. `yarn install`
2. `yarn audit`: expected 0 vulnerabilities
3. `yarn start [OPTIONS]` to test CLI

## Impacted Areas in app

* dependencies
